src/Kunstmaan/ConfigBundle/README.md

```diff
@@ -0,0 +1,95 @@
+# KunstmaanConfigBundle 
```

Comment:
move this content to the docs directory + add link from this README to the documentation. This way we can add it to the bundle site